py/client/pydeephaven/session.py

```diff
@@ -449,45 +449,46 @@ def open_table(self, name: str) -> Table:
                 faketable.ticket = None
                 faketable.schema = None
 
-    def publish_table(self, table: Table, shared_ticket: SharedTicket) -> None:
-        """Publishes a table with the given shared ticket for sharing with other sessions.
+    def publish_table(self, ticket: SharedTicket, table: Table) -> None:
+        """Publishes a table to the given shared ticket. The ticket can then be used by another session to fetch the
+        table.
```

Comment:
We should eventually support other types here besides just SharedTicket. Just noting that I expect we'll need some sort of base Ticket class in the future and the docs will need to be updated.

Reply:
Yes, and the same may need to happen in the client too.